### PR TITLE
[TT-11018, DX-977]add API templates docs

### PR DESCRIPTION
**Pull request by jeffy-mathew**
### For internal users - Please add a Jira DX PR ticket to the subject!
https://tyktech.atlassian.net/browse/TT-11018

---

### Preview Link
https://deploy-preview-3942--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview/

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/tyk-apis/tyk-dashboard-api/templates.md [hunk 1]
```diff
@@ -0,0 +1,602 @@
+# API Templates
+
+## Overview
+
+API Templates is a new feature designed to streamline the process of defining APIs using the OpenAPI Specification (OAS). These templates serve as a foundational or default structure for various segments of an OAS API definition, enhancing the efficiency and consistency of API creation and importation.
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-apis/tyk-dashboard-api/templates.md]:
```suggestion
API Templates are a new feature designed to streamline the process of defining APIs using the OpenAPI Specification (OAS). These templates provide a default structure for various segments of an OAS API definition, enhancing the efficiency and consistency of API creation and importation.
```

File: tyk-docs/content/tyk-apis/tyk-dashboard-api/templates.md [hunk 2]
```diff
@@ -0,0 +1,602 @@
...
+
+## Key Features
+
+- **Starting Point for API Definitions**: API templates provide a base structure for OAS API definitions, ensuring uniformity and saving time in the API creation process.
+- **Application during Import or Creation**: Templates can be applied both when importing an existing API or during the creation of a new one.
+- **Easy Identification through IDs**: Templates are identifiable either through a database-generated ID or a human-readable ID, facilitating easy reference.
+
+## Creating an API Template
+
+To create an API template use `/api/assets` dashboard endpoint to create an Asset of kind `oas-template`.
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-apis/tyk-dashboard-api/templates.md]: Should maybe add a link to the endpoint here when available, or add a link to the Dashboard OAS API?

```suggestion
To create an API template use the `/api/assets` dashboard endpoint to create an *Asset* of kind `oas-template`.
```

File: tyk-docs/content/tyk-apis/tyk-dashboard-api/templates.md [hunk 3]
```diff
@@ -0,0 +1,602 @@
...
+
+## Applying Templates
+
+### During API Import
+
+1. **Specify Template ID**: Use the `templateID` parameter in the request. This can be the database ID or the human-readable ID of the template.
+2. **Merge Process**: The request payload merges into the selected template. 
+    * For maps like OAS paths and different components, keys are added alongside existing ones in the template, if there are any key already existing in the template, they are replaced (request payload has precedence).
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-apis/tyk-dashboard-api/templates.md]:
```suggestion
    - For maps like OAS paths and different components, keys are added alongside existing ones in the template, if a key already exists in the template, it is replaced (request payload has precedence).
```

File: tyk-docs/content/tyk-apis/tyk-dashboard-api/templates.md [hunk 4]
```diff
@@ -0,0 +1,602 @@
...
+
+```
+curl --location 'http://tyk-dashboard:3000/api/assets' \
+--header 'Authorization: d244265dd51a4db55b77fad200198d85' \
+--header 'Content-Type: application/json' \
+--header 'Cookie: csrf_token=EWtTyjTrZoD0/cuFqsNPKcerjkqUGDTEMvdui4mHVFM=' \
+--data '{
+    "kind": "oas-template",
+    "id": "healthcheck",
+    "data": {
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-apis/tyk-dashboard-api/templates.md]: So the data payload of an oas-template is the schema. Maybe this should be mentioned firstly in the introduction overview before a full example is given. Could we make the example collapsible using summary detail html tags to reduce example length?

File: tyk-docs/content/tyk-apis/tyk-dashboard-api/templates.md [hunk 5]
```diff
@@ -0,0 +1,602 @@
...
+                        }
+                    }
+                }
+            }
+        }
+    }
+}'
+```
+
+Now, let's use this template while importing an OAS document. Let's use a petstore OAS example here.
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-apis/tyk-dashboard-api/templates.md]: Not sure if my understanding is correct so checking

When the oas-template is applied when importing an OAS document, then the endpoints and x-tyk-api-gateway definition are applied to the target OAS API? 

If so, then could mention/expand upon this in the introduction/overview section, before the reader gets to the example.

The benefit is that this same template can be applied to multiple OAS APIs, applying the endpoint paths in and x-tyk-api-gateway definition to the template.

File: tyk-docs/content/tyk-apis/tyk-dashboard-api/templates.md [hunk 6]
```diff
@@ -0,0 +1,602 @@
...
+          "ignoreAuthentication": {
+            "enabled": true
+          }
+        }
+      }
+    }
+  }
+}
+```
+
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-apis/tyk-dashboard-api/templates.md]: Summarise what the example has done here. Maybe make the example content expandable/collapsible using _html_ summary and _detail_ tags in the markdown

**Review by dcs3spp — CHANGES_REQUESTED** [review]
PR LGTM, suggested minor changes

File: tyk-docs/content/tyk-apis/tyk-dashboard-api/templates.md (hunk 1 above)
Comment (titpetric) [reply tyk-docs/content/tyk-apis/tyk-dashboard-api/templates.md]: API Templates *are* a new feature...?

**Comment by titpetric** [comment]
Some plain language comments raised with write-good (vale):

```
 templates.md
 5:15    suggestion  Try to avoid using 'is'.        write-good.E-Prime  
 10:60   warning     'be applied' may be passive     write-good.Passive  
                     voice. Use active voice if you                      
                     can.                                                
 10:60   suggestion  Try to avoid using 'be'.        write-good.E-Prime  
 11:50   suggestion  Try to avoid using 'are'.       write-good.E-Prime  
 21:85   suggestion  Try to avoid using 'be'.        write-good.E-Prime  
 23:62   suggestion  Try to avoid using 'are'.       write-good.E-Prime  
 23:62   warning     'are added' may be passive      write-good.Passive  
                     voice. Use active voice if you                      
                     can.                                                
 23:122  suggestion  Try to avoid using 'are'.       write-good.E-Prime  
 23:134  warning     'already existing' is too       write-good.TooWordy 
                     wordy.                                              
 23:173  suggestion  Try to avoid using 'are'.       write-good.E-Prime  
 23:173  warning     'are replaced' may be passive   write-good.Passive  
                     voice. Use active voice if you                      
                     can.                                                
 25:68   warning     'it is' is too wordy.           write-good.TooWordy 
 25:71   suggestion  Try to avoid using 'is'.        write-good.E-Prime  
 27:290  suggestion  Try to avoid using 'is'.        write-good.E-Prime  
 596:88  suggestion  Try to avoid using 'being'.     write-good.E-Prime  
 600:13  suggestion  Try to avoid using 'are'.       write-good.E-Prime  
 600:28  warning     'only' is a weasel word!        write-good.Weasel   
 600:92  suggestion  Try to avoid using 'be'.        write-good.E-Prime  
 600:92  warning     'be applied' may be passive     write-good.Passive  
                     voice. Use active voice if you                      
                     can.                                                

✖ 0 errors, 7 warnings and 12 suggestions in 1 file.
```

I'd say out of this the main issue is that we should be using active voice, and reword some things like `how many items returned` to `the number of items returned` for clarity.

Do we have a better strategy for providing the example jsons? Can we stick that into it's own files and just link them? Use `curl -d @file.json` in the examples? It makes no sense to inline the full assets into the docs.

**Comment by andyo-tyk** [comment]
I have reworked this PR - @jeffy-mathew @titpetric @dcs3spp please would you review again? 🙏 
The starting point for the preview is here: https://deploy-preview-3942--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview/

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+#### Standardising API configuration
+If you have specific requirements for your external facing APIs - for example authentication, caching or even a healthcheck endpoint - you can define the appropriate API templates so that when APIs are created on Tyk these fields are automatically and correctly configured.
+
+## How API templating works
+An API template is a blueprint from which you can build new APIs - it is an incomplete Tyk OAS API definition JSON that you can use as the starting point when creating a new API on Tyk. There is no limit to how much or how little of the API definition is pre-configured in the template (such that when you choose to create a new API without choosing a template, the blank API definition that you start from is itself a template).
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview.md]:
```suggestion
An API template is a blueprint from which you can build new APIs - it is an incomplete JSON representation of a Tyk OAS API definition that you can use as the starting point when creating a new API on Tyk. There is no limit to how much or how little of the API definition is pre-configured in the template (such that when you choose to create a new API without choosing a template, the blank API definition that you start from is itself a template).
```

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-designer.md [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+
+The Tyk Dashboard UI provides the following functionality to support working with API templates:
+ - Creating templates
+   - [new template]({{< ref "product-stack/tyk-dashboard/advanced-configurations/templates/template-designer#creating-a-new-api-template" >}})
+   - [from an existing API]({{< ref "product-stack/tyk-dashboard/advanced-configurations/templates/template-designer#creating-a-template-from-an-existing-api" >}})
+ - Using templates
+   - [when creating an API]({{< ref "product-stack/tyk-dashboard/advanced-configurations/templates/template-designer#using-a-template-when-creating-a-new-api" >}})
+   - [when importing an OpenAPI description or API definition]({{< ref "product-stack/tyk-dashboard/advanced-configurations/templates/template-designer#using-a-template-when-importing-an-openapi-description-or-api-definition" >}})
+ - [Managing templates]({{< ref "product-stack/tyk-dashboard/advanced-configurations/templates/template-designer#" >}})
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-designer.md]:
```suggestion
 - [Managing templates]({{< ref "product-stack/tyk-dashboard/advanced-configurations/templates/template-designer#managing-templates" >}})
```

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-designer.md [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+Or, from the **Created APIs** screen, select **ADD NEW API** as normal and then select the template you want to use from the **API Template** section:
+{{< img src="/img/dashboard/api-assets/api-templates/create-api-from-template2.png" alt="Select the template you want to use" >}}
+
+Both of these routes will take you through to the API Designer, where the settings from your API template will be pre-configured.
+
+In this example, we applied "My first template" that we created [here]({{< ref "product-stack/tyk-dashboard/advanced-configurations/templates/template-designer#creating-a-new-api-template" >}}). You can see that the Gateway Status and Access fields have been configured:
+{{< img src="/img/dashboard/api-assets/api-templates/created-api.png" alt="The API with template applied" >}}
+
+### Using a template when importing an OpenAPI description or API definition
+From the **Import API** screen, if you select the OpenAPI **type** then you can create an API from an OpenAPI description or Tyk OAS API definition that you paste into the API Designer, or that is retrieved from a file that is either uploaded to Tyk Dashboard or accessed from a URL.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-designer.md]:
```suggestion
From the **Import API** screen, if you select the OpenAPI **type** then you can create an API using:
- OpenAPI description or Tyk OAS API definition that you paste into the API Designer
- A file that is either uploaded to Tyk Dashboard or accessed from a URL
```

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-designer.md [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+This will take you to the **API Template Details** screen where you can view the current template configuration. If you want to make changes, simply select **EDIT** to make the fields editable:
+{{< img src="/img/dashboard/api-assets/api-templates/template-editor.png" alt="Modifying the API template" >}} 
+
+Alternatively you can view and modify the raw JSON for the template by selecting **VIEW RAW TEMPLATE** from the **ACTIONS** menu:
+{{< img src="/img/dashboard/api-assets/api-templates/template-raw-editor.png" alt="Modifying the API template JSON" >}} 
+
+You'll need to select **SAVE TEMPLATE** to apply your changes from either screen.
+
+### Deleting a template
+You can delete a template from your Tyk Dashboard from the **API Template Details** screen, which you can access by selecting the template from the **API Templates** screen (either by clicking on the template name, or selecting **EDIT TEMPLATE** from the **ACTIONS** menu):
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-designer.md]:
```suggestion
You can delete a template from your Tyk Dashboard from the **API Template Details** screen. This screen can be accessed by selecting the template from the **API Templates** screen (either by clicking on the template name, or selecting **EDIT TEMPLATE** from the **ACTIONS** menu):
{{< img src="/img/dashboard/api-assets/api-templates/edit-template.png" alt="Accessing the API template" >}} 
```

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-api.md [hunk 11]
```diff
@@ -0,0 +1,403 @@
...
+
+{{< note success >}}
+**Note**  
+
+API Templates are exclusive to [Tyk OAS APIs]({{< ref "getting-started/key-concepts/what-is-an-api-definition#api-definition-types" >}}).
+{{< /note >}}
+
+## Structure of an API template
+An API template asset has the following structure:
+ - `id`: a unique string type identifier for the template
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-api.md:23]:
```suggestion
 - `id`: a unique string type identifier for the template that can be autogenerated or specified by the user
```

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview.md [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+
+Templates are used only during the creation of an API, they cannot be applied later. Before you can use a template as the basis for an API, you must register the template with Tyk Dashboard.
+
+### Structure of an API template
+An API template asset has the following structure:
+ - `id`: a unique string type identifier for the template
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview.md:32]:
```suggestion
- `id`: a unique string type identifier for the template that can be autogenerated or specified by the user
```

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview.md [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+ - `data`: a Tyk OAS API definition, the content of which will be used for templating APIs
+ - `DBID`: a unique identifier assigned by Tyk when the template is registered in the Dashboard database
+
+### Creating an API from a template
+When you use a template during the [creation]({{< ref "getting-started/using-oas-definitions/create-an-oas-api" >}}) of an API, the fields configured in `data` will be pre-set in your new API. You are able to modify these during and after creation of the template. No link is created between the API and the template, so changes made to the API will not impact the template.
+
+### Merging with an OpenAPI description or Tyk OAS API definition
+When you use a template during the creation of an API where you [import]({{< ref "getting-started/using-oas-definitions/import-an-oas-api" >}}) the OpenAPI document or a full Tyk OAS API definition, the template is used as the starting point with the imported description merged over the top. If the `x-tyk-api-gateway` extension exists in the template, it will be applied to the newly created API.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview.md]:
```suggestion
When you use a template during the creation of an API where you [import]({{< ref "getting-started/using-oas-definitions/import-an-oas-api" >}}) the OpenAPI document or a full Tyk OAS API definition, the template is combined with the imported OAS description. If the `x-tyk-api-gateway` extension exists in the template, it will be applied to the newly created API.
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Thanks for updating the API templates docs @andyo-tyk , I have submitted second review with minor suggestions

**Comment by dcs3spp** [comment]
> I have reworked this PR - @jeffy-mathew @titpetric @dcs3spp please would you review again? 🙏 The starting point for the preview is here: https://deploy-preview-3942--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview/

@andyo-tyk I've submitted a second review

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview.md [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+ - `kind`: the asset type, which is set to `oas-template`
+ - `name`: human-readable name for the template
+ - `description`: a short description of the template, that could be used for example to indicate the configuration held within the template
+ - `data`: a Tyk OAS API definition, the content of which will be used for templating APIs
+ - `DBID`: a unique identifier assigned by Tyk when the template is registered in the Dashboard database
+
+### Creating an API from a template
+When you use a template during the [creation]({{< ref "getting-started/using-oas-definitions/create-an-oas-api" >}}) of an API, the fields configured in `data` will be pre-set in your new API. You are able to modify these during and after creation of the template. No link is created between the API and the template, so changes made to the API will not impact the template.
+
+### Merging with an OpenAPI description or Tyk OAS API definition
```
Comment (jeffy-mathew) [inline tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview.md:42]: Shall we name it `OpenAPI document` instead of `OpenAPI description`?

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-api.md [hunk 15]
```diff
@@ -0,0 +1,403 @@
...
+
+## Applying a template when creating an API from an OpenAPI document
+When creating an API on Tyk using an OpenAPI document describing your upstream service, you can use the `/apis/oas/import` endpoint to import the OpenAPI description and apply it to your API.
+
+If you have a template registered with your Dashboard, you can use this as the starting point for your new API. Tyk will combine the OpenAPI document with the template, automating the configuration of any element in the Tyk OAS API definition as defined in your chosen template.
+
+You'll need to identify the template to be used during the import. You can use either its unique `id` or the database ID that was assigned when the template was [registered with Tyk Dashboard]({{< ref "product-stack/tyk-dashboard/advanced-configurations/templates/template-api#registering-a-template-with-tyk-dashboard" >}}). You provide either the `id` or `DBID` in the `templateID` query parameter in the call to `/oapis/oas/import`.
```
Comment (jeffy-mathew) [inline tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-api.md]:
```suggestion
You'll need to identify the template to be used during the import. You can use either its unique `id` or the database ID that was assigned when the template was [registered with Tyk Dashboard]({{< ref "product-stack/tyk-dashboard/advanced-configurations/templates/template-api#registering-a-template-with-tyk-dashboard" >}}). You provide either the `id` or `_id ` in the `templateID` query parameter in the call to `/oapis/oas/import`.
```

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-api.md [hunk 16]
```diff
@@ -0,0 +1,403 @@
...
+{{< /note >}}
+
+## Structure of an API template
+An API template asset has the following structure:
+ - `id`: a unique string type identifier for the template
+ - `kind`: the asset type, which is set to `oas-template`
+ - `name`: human-readable name for the template
+ - `description`: a short description of the template, that could be used for example to indicate the configuration held within the template
+ - `data`: a Tyk OAS API definition, the content of which will be used for templating APIs
+ - `DBID`: a unique identifier assigned by Tyk when the template is registered in the Dashboard database
```
Comment (jeffy-mathew) [inline tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-api.md]:
```suggestion
 - `_id`: a unique identifier assigned by Tyk when the template is registered in the Dashboard database
```

`_id` is the field that is used in json response for DB ID

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview.md [hunk 17]
```diff
@@ -0,0 +1,54 @@
...
+Templates are used only during the creation of an API, they cannot be applied later. Before you can use a template as the basis for an API, you must register the template with Tyk Dashboard.
+
+### Structure of an API template
+An API template asset has the following structure:
+ - `id`: a unique string type identifier for the template
+ - `kind`: the asset type, which is set to `oas-template`
+ - `name`: human-readable name for the template
+ - `description`: a short description of the template, that could be used for example to indicate the configuration held within the template
+ - `data`: a Tyk OAS API definition, the content of which will be used for templating APIs
+ - `DBID`: a unique identifier assigned by Tyk when the template is registered in the Dashboard database
```
Comment (jeffy-mathew) [inline tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview.md]:
```suggestion
 - `_id`: a unique identifier assigned by Tyk when the template is registered in the Dashboard database
```

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-api.md [hunk 18]
```diff
@@ -0,0 +1,403 @@
...
+
+## Applying a template when creating an API from a Tyk OAS API definition
+When creating an API using a complete Tyk OAS API definition (which includes `x-tyk-api-gateway`), you can use the `/apis/oas` endpoint to import the API defintiion.
+
+If you have a template registered with your Dashboard, you can use this as the starting point for your new API. Tyk will combine the API definition with the template, automating the configuration of any element defined in your chosen template.
+
+You'll need to identify the template to be used during the import. You can use either its unique `id` or the database ID that was assigned when the template was [registered with Tyk Dashboard]({{< ref "product-stack/tyk-dashboard/advanced-configurations/templates/template-api#registering-a-template-with-tyk-dashboard" >}}). You provide either the `id` or `DBID` in the `templateID` query parameter in the call to `/apis/oas`.
```
Comment (jeffy-mathew) [inline tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-api.md]:
```suggestion
You'll need to identify the template to be used during the import. You can use either its unique `id` or the database ID that was assigned when the template was [registered with Tyk Dashboard]({{< ref "product-stack/tyk-dashboard/advanced-configurations/templates/template-api#registering-a-template-with-tyk-dashboard" >}}). You provide either the `id` or `_id` in the `templateID` query parameter in the call to `/apis/oas`.
```

**Comment by dcs3spp** [comment]
@andy I've resolved most comments apart from two suggestion comments [https://github.com/TykTechnologies/tyk-docs/pull/3942#discussion_r1516105273](https://github.com/TykTechnologies/tyk-docs/pull/3942#discussion_r1516105273) and [https://github.com/TykTechnologies/tyk-docs/pull/3942#discussion_r1516101045](https://github.com/TykTechnologies/tyk-docs/pull/3942#discussion_r1516101045)

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-designer.md (hunk 9 above)
Comment (andyo-tyk) [reply tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-designer.md]:
```suggestion
From the **Import API** screen, if you select the OpenAPI **type** then you can create an API from an OpenAPI description or Tyk OAS API definition; choose the appropriate method to provide this to the Dashboard:
- paste the JSON into the text editor
- provide a plain text file containing the JSON
- provide a URL to the JSON
```

File: tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview.md (hunk 14 above)
Comment (andyo-tyk) [reply tyk-docs/content/product-stack/tyk-dashboard/advanced-configurations/templates/template-overview.md:42]: No, this is deliberate: the description is the JSON, the document is the file containing the description

**Review by andyo-tyk — COMMENTED** [review]
Improved wording

**Review by andyo-tyk — APPROVED** [review]

**Review by dcs3spp — APPROVED** [review]
